html/pfappserver/lib/pfappserver/Model/Search/Node.pm

```diff
@@ -95,8 +95,9 @@ sub make_builder {
             L_("IFNULL(node_category.name, '')", 'category'),
             L_("IFNULL(node_category_bypass_role.name, '')", 'bypass_role'),
             L_("IFNULL(device_class, ' ')", 'dhcp_fingerprint'),
+            L_("IF(locationlog.end_time = '0000-00-00 00:00:00', 'on' ,'off')", 'online'),
             { table => 'iplog', name => 'ip', as => 'last_ip' },
-            { table => 'locationlog', name => 'switch', as => 'switch_ip' }
+            { table => 'locationlog', name => 'switch', as => 'switch_id' }
```

Comment:
I assume we would want it since we may be searching for : 
`switch_mac LIKE '01:23:%'`

Then we would only see the switch ID which could be the IP if it is a controller